Gemfile

```diff
@@ -139,8 +139,8 @@ source "https://rails-assets.org" do
   gem "rails-assets-jquery-ujs",                      "~>1.1.0"
   gem "rails-assets-jqueryui",                        "~>1.9.2"
   gem "rails-assets-bootstrap-switch",                "~>3.3.2"
-  gem "rails-assets-angular-bootstrap-switch",        "~>0.4.1"
-  gem "rails-assets-kubernetes-topology-graph",       "= 0.0.22"
+  gem "rails-assets-romanblanco--angular-bootstrap-switch",        "~>0.4.2"
+  gem "rails-assets-romanblanco--topology-graph",       "= 0.0.23"
```

Comment:
@romanblanco, https://github.com/kubernetes-ui/topology-graph/pull/18 is merged,  and I rebuilt rails-assets to have 0.0.23 available, so you can change that part here.